frontend/package.json

```diff
@@ -16,7 +16,10 @@
     "react": "^18.0.0",
     "react-burger-menu": "^3.0.8",
     "react-dom": "^18.0.0",
+    "react-feather": "^2.0.10",
```

Comment:
Feather is included in Icons, you can remove this line

Reply:
ok :)